<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateInforNewDialog</class>
 <widget class="QDialog" name="UpdateInforNewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#UpdateInforNewDialog{
	border-radius: 4px;
	background-color: rgb(255, 255, 255, 254);
	background-position: top left right bottom;
	background-clip: padding;
	border: 1px solid #d7d8d9;
}

QWidget
{
	font-family:'微软雅黑';
}

QPushButton#pushButton_once{
	border-radius: 0px;
    color:#FFFFFF;
    background-color:#3DCE3D;
}
QPushButton#pushButton_once:hover{
background-color:#38C738;
}

QPushButton#pushButton_once:pressed{
	background-color:#2FB82F;
}

QPushButton#pushButton_later{
    color:#666666;
    border:1px solid #E5E5E5;
}
QPushButton#pushButton_later:hover{
    background-color:#f5f5f5;
}
QPushButton#pushButton_later:pressed{
    background-color:#EBEBEB;
}

QLabel#label_Infor
{
	color: #7A7A7A;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="dialogbg" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="TitlebarWidget" name="Title_bar" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_Content" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>50</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>50</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_msg">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Infor">
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>为了不影响您的业务咨询，强烈要求您升级后继续使用，否则影响后续使用</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_btn" native="true">
           <property name="minimumSize">
            <size>
             <width>550</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>550</width>
             <height>60</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>12</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>15</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>412</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_later">
              <property name="minimumSize">
               <size>
                <width>115</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>140</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>暂不升级,继续使用</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_once">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>升级使用</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitlebarWidget</class>
   <extends>QWidget</extends>
   <header location="global">TitlebarWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
